app/components/AsideBar.js

```diff
@@ -2,32 +2,35 @@ import React, { PropTypes } from 'react';
 import ClosestCities from './ClosestCities';
 import css from '../styles/aside.scss';
 
-// temporary data filler
-const closestCities = [
-    'City1',
-    'City2',
-    'City3',
-    'City4',
-    'City5',
-];
-
-function AsideBar({ className }) {
+function AsideBar({ className, nearestCities }) {
     return (
         <aside className={className}>
-            <ClosestCities
-              title={(<h3>Closest cities to your current location</h3>)}
-              list={closestCities}
-            />
+            {
+                nearestCities.length === 0
+                ? (
+                    <div className="error-container">
+                        <h3>No cities found</h3>
+                    </div>
+                )
+                : (
+                    <ClosestCities
+                      title={<h3>Nearest cities to your current location</h3>}
+                      list={nearestCities}
+                    />
```

Comment:
the codestyle is taken from here:
[https://github.com/airbnb/javascript/tree/master/react#user-content-alignment](url)
eslint is set up accordingly to this styleguide